workers/unity/Assets/Fps/Scripts/SetupLogic/iOSWorkerConnector.cs

```diff
@@ -0,0 +1,177 @@
+using System.Collections.Generic;
+using System.Text.RegularExpressions;
+using System.Threading.Tasks;
+using Improbable.Gdk.Core;
+using Improbable.Gdk.GameObjectCreation;
+using Improbable.Gdk.GameObjectRepresentation;
+using Improbable.Gdk.Mobile;
+using Improbable.Gdk.PlayerLifecycle;
+using Improbable.Worker.CInterop;
+using UnityEngine;
+#if UNITY_IOS
+using Improbable.Gdk.Mobile.iOS;
+
+#endif
+
+namespace Fps
+{
+    [RequireComponent(typeof(ConnectionController))]
+    public class iOSWorkerConnector : MobileWorkerConnector, ITileProvider
+    {
+        private const string AuthPlayer = "Prefabs/MobileClient/Authoritative/Player";
+        private const string NonAuthPlayer = "Prefabs/MobileClient/NonAuthoritative/Player";
+
+        private const string Small = "small";
+        private const string Large = "large";
+
+        public string forcedIpAddress;
+
+        public int TargetFrameRate = 60;
+
+        public GameObject SmallLevelPrefab;
+        public GameObject LargeLevelPrefab;
+
+        private GameObject levelInstance;
+
+        private ConnectionController connectScreen;
+
+        private List<TileEnabler> levelTiles = new List<TileEnabler>();
+        public List<TileEnabler> LevelTiles => levelTiles;
+
+        private ConnectionController connectionController;
+
+        public string IpAddress { get; set; }
+
+        private void OnValidate()
+        {
+            forcedIpAddress = Regex.Replace(forcedIpAddress, "[^0-9.]", "");
+        }
+
+        public async void TryConnect()
+        {
+            await Connect(WorkerUtils.iOSClient, new ForwardingDispatcher()).ConfigureAwait(false);
+        }
+
+        private void Awake()
+        {
+            connectionController = GetComponent<ConnectionController>();
+        }
+
+        protected virtual async void Start()
+        {
+            Application.targetFrameRate = TargetFrameRate;
+            IpAddress = forcedIpAddress;
+
+            await AttemptConnect();
+        }
+
+        protected override string GetHostIp()
+        {
+#if UNITY_IOS
+            if (!string.IsNullOrEmpty(IpAddress))
+            {
+                return IpAddress;
+            }
+
+            if (Application.isMobilePlatform && iOSDeviceInfo.ActiveDeviceType == MobileDeviceType.Virtual)
+            {
+                // TODO What should this return? No "EmulatorDefaultCallbackIp" in iOSDeviceInfo
```

Comment:
iOS sim Ip is the same as local machine, so we can just return RuntimeConfigDefaults.ReceptionistHost.
We probably don't need this if

Reply:
@gkassabli this comment seems truncated

Reply:
Haha, it wasn't, but ending the comment with if and without period is indeed confusing :)